internal/gps/result_test.go

```diff
@@ -35,12 +34,6 @@ func init() {
 			}, nil),
 		},
 	}
-
-	// just in case something needs punishing, kubernetes is happy to oblige
-	kub = atom{
-		id: pi("github.com/kubernetes/kubernetes"),
-		v:  NewVersion("1.0.0").Is(Revision("528f879e7d3790ea4287687ef0ab3f2a01cc2718")),
```

Comment:
Commenting it out is also OK, I suppose.

The biggest reason to pick a revision is just the chaos that results from suddenly this, and having people pick their own random revisions. Or, worse, a branch tip.

Basically, this is me just trying to proactively head off a coordination problem, should it arise.